webapp/views/wpt-results.js

```diff
@@ -861,6 +879,41 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
     this.path = this.searchResults[(n + next) % n].test;
   }
 
+  sortTestname() {
+    if (!this.displayedNodes) {
+      return;
+    }
+    const sortedNodes = this.displayedNodes.slice();
+    sortedNodes.sort(function (a, b) {
+      if (a.path < b.path) {
+        return -1;
+      }
+
+      if (a.path > b.path) {
+        return 1;
+      }
+      return 0;
+    })
+
+    this.displayedNodes = sortedNodes;
+  }
+
+  sortTestResults() {
+    const [index] = arguments;
+    return () => {
+      if (!this.displayedNodes) {
+        return;
+      }
+
+      const sortedNodes = this.displayedNodes.slice();
+      sortedNodes.sort(function (a, b) {
+        return a.results[index].passes - b.results[index].passes;
```

Comment:
https://github.com/web-platform-tests/wpt.fyi/pull/2854#issuecomment-1131828262. I think we decide to go with percentage since it will be the new display. The secondary sort is based on test names.